starcheck/src/aca_load_review_cl.rst

```diff
@@ -236,9 +234,12 @@ Checks
 |ACA-021|ACA field-of-view |X|X|FL: Z at least half-width inside           |n/a |Reduced aspect  |
 |       |limits            | | |field-of-view limits                       |    |quality         |
 +-------+------------------+-+-+-------------------------------------------+----+----------------+
-|       |                  |X|X|AS: spoiled by another object brighter than|    |Possible Bright |
-|ACA-022|Spoiler stars     | | |mag(AS) + 0.2, that lies closer than       |n/a |Star Hold       |
-|       |                  | | |maneuver uncertainty to the AS search box  |    |                |
+|       |                  | | |AS: Impact of spoilers on acquisition      |    |                |
+|       |                  | | |probability is calculated in proseco.      |    |                |
+|ACA-022|Spoiler stars     |X|X|Previously this was handled as a red warn  |n/a |Possible Bright |
```

Comment:
It seems like if we define the checklist as a set of checks that *must* be handled by our load review tool, that ACA-022 could stay in the checklist with the new text "Impact of acquisition spoilers included in acquisition probability calculation" (that would remove proseco -- which seemed to confuse some things as a star *selection* tool).

Reply:
> "Impact of acquisition spoilers included in acquisition probability calculation"

This looks good, let's do it.